docs/development/extensions-core/lookups-cached-global.md

```diff
@@ -214,12 +214,12 @@ The remapping values for each globally cached lookup can be specified by a JSON
 |Property|Description|Required|Default|
 |--------|-----------|--------|-------|
 |`pollPeriod`|Period between polling for updates|No|0 (only once)|
-|`uri`|URI for the file of interest, specified as a file, hdfs, or s3 path|No|Use `uriPrefix`|
+|`uri`|URI for the file of interest, specified as a file, hdfs, s3 or gs path|No|Use `uriPrefix`|
```

Comment:
@clintropolis not sure if this should be kept `gs` or `gcs`

Reply:
hmm, I think this one is fine to leave as is